packages/desktop-client/src/components/AppBackground.tsx

```diff
@@ -21,7 +21,7 @@ function AppBackground({ initializing, loadingText }: AppBackgroundProps) {
 
       {(loadingText != null || initializing) && (
         <View
-          {...css({
+          className={`${css({
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I wonder what's better. This vs `String(css(..))`.

Reply:
I think it doesn't matter much as this will only be there until we transition to emotionjs